### ISSUE TITLE
fails to compile for 386?

### ISSUE THREAD

**Issue by mika** [aelsabbahy/goss#22]
While amd64 works fine:

````
% GO15VENDOREXPERIMENT=1 GOOS=linux GOARCH=amd64 go build -ldflags "-X main.version="0.0.0"" -o release/goss-linux-amd64 github.com/aelsabbahy/goss/cmd/goss
% 
````

... compiling for 386 fails for me:

````
% GO15VENDOREXPERIMENT=1 GOOS=linux GOARCH=386 go build -ldflags "-X main.version="0.0.0"" -o release/goss-linux-386 github.com/aelsabbahy/goss/cmd/goss
# github.com/aelsabbahy/goss/util/group
/grml/gostuff/src/github.com/aelsabbahy/goss/util/group/lookup.go:5: undefined: currentGroup
/grml/gostuff/src/github.com/aelsabbahy/goss/util/group/lookup.go:17: cannot use gid (type string) as type int in argument to lookupGroupID
/grml/gostuff/src/github.com/aelsabbahy/goss/util/group/lookup.go:27: undefined: groupMembers
````

This might be totally my fault, but maybe you've an hint/idea?

Thanks for goss! :)

**Comment by aelsabbahy** [comment]
Never tried to compile it for 386.

I believe I copied go-group (and modified it slightly) from https://github.com/danryan/go-group

Does that compile using GOARCH=386 for you?

Glad you're finding goss useful :)

**Comment by mika** [comment]
Nope, same problem seems to be there.

**Comment by mika** [comment]
Alright, compiling for 386 on a 386 system works though, the problem is the cross compiling (so building for 386 e.g. on a amd64 system), I assume caused by the inline C code.

**Comment by aelsabbahy** [comment]
Fixed in [v0.0.22](https://github.com/aelsabbahy/goss/releases/tag/v0.0.22)

Let me know if you have any issues.
